### ISSUE TITLE
Invalid middleware doctrine_clear_entity_manager 

### ISSUE THREAD

**Issue by dpinheiro** [doctrine/DoctrineBundle#1051]
Hi folks

Was just trying to update my symfony 4.4 to the beta-2 and just bumped into this problem where we can no longer use the doctrine_clear_entity_middleware

In my previous configuration was using the `doctrine_clear_entity_manager` middleware, but after updating to the new beta, due to the [this commit](https://github.com/doctrine/DoctrineBundle/commit/21846c47c166ca86762135cd932a977b4309414e) where the name of the service was changed from `messenger.middleware.doctrine_clear_entity_manager` to `doctrine.orm.messenger.event_subscriber.doctrine_clear_entity_manager` the service can no longer be found as the MessengerPass is trying to prefix it with `messenger.middleware` as can be seen [here](https://github.com/symfony/messenger/blob/39444584eaec8a351da331e519d705931aa97235/DependencyInjection/MessengerPass.php#L312-L335)

This middleware can still be used referring to it's full alias `doctrine.orm.messenger.event_subscriber.doctrine_clear_entity_manager` but is this the desirable behaviour? There's any reason why the service name was changed? Should we update the `MessengerPass` to also check for this new prefix `doctrine.orm.messenger.event_subscriber`?

@weaverryan maybe you can give some feedback here :)

**Comment by dpinheiro** [comment]
Just one update, even using the alias `doctrine.orm.messenger.event_subscriber.doctrine_clear_entity_manager` this still not works because `DoctrineClearEntityManagerWorkerSubscriber` does not implement the `MiddlewareInterface`

`TypeError : Return value of Symfony\Component\Messenger\Middleware\MiddlewareStack::next() must implement interface Symfony\Component\Messenger\Middleware\MiddlewareInterface or be null, instance of Symfony\Bridge\Doctrine\Messenger\DoctrineClearEntityManagerWorkerSubscriber returned`

**Comment by alcaeus** [comment]
The `doctrine_clear_entity_manager` has been dropped in #1043 in favour of the `doctrine.orm.messenger.event_subscriber.doctrine_clear_entity_manager` event subscriber. This subscriber is registered automatically, so you don't need to add anything. In your case, you need to remove the `doctrine_clear_entity_manager` middleware from the configuration and everything should work.

If I may ask, did you add the middleware recently, and if so, based on what information? We've had it in 1.12.x-dev for a while but removed it before the beta release since the corresponding class was dropped from Symfony 4.4 during the development phase.

**Comment by dpinheiro** [comment]
Thanks for the feedback. I updated my code to 4.4-dev some weeks ago in order to be able to use the `doctrine_clear_entity_manager`, didn't knew it has been set as the default. If I'm not wrong I just saw it on [this blog post](https://symfony.com/blog/new-in-symfony-4-4-messenger-middleware-to-clear-doctrine-entity-manager). Just did a quick check around and don't see it anywhere in the documentation so we should be good to go, this can be closed
